### PR TITLE
Pract.03: Add 02 | Ivan Angelov

### PR DESCRIPTION
**Pull request by ivan-angelov9643**
Ivan Angelov
SI 5 9MI0600190

### CODE REVIEW

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (Justsvetoslavov) [inline Seminars/Sem.03/Pract.03/Task02.cpp]: +1 for null terminator, do we have padding ? 👀

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (Justsvetoslavov) [inline Seminars/Sem.03/Pract.03/Task02.cpp]: It isn't that long to separate on x2 lines

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (Justsvetoslavov) [inline Seminars/Sem.03/Pract.03/Task02.cpp]: It isn't that long to separate on x2 lines

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (Justsvetoslavov) [inline Seminars/Sem.03/Pract.03/Task02.cpp]: Avoid return -1?
Waiting on an answer why we shouldn't return -1 in main function

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (Justsvetoslavov) [inline Seminars/Sem.03/Pract.03/Task02.cpp:50]: too many empty lines.

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (Justsvetoslavov) [inline Seminars/Sem.03/Pract.03/Task02.cpp]: const ?

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (Justsvetoslavov) [inline Seminars/Sem.03/Pract.03/Task02.cpp]: why is fileSize here?

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]
I have noted some of the mistakes.
Please re-visit your solution, test it and apply fixes.
Then tag me for a second review.

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (ivan-angelov9643) [reply Seminars/Sem.03/Pract.03/Task02.cpp]: edit: my bad you meant char name[MAX_COMPANY_NAME_LENGTH + 1];

There are 3 bytes padding after the last element of the companyName variable followed by 4 bytes for the teamCount variable.

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (ivan-angelov9643) [reply Seminars/Sem.03/Pract.03/Task02.cpp]: Returning -1 from main() is not a good practice because it is not guaranteed to be recognized as an error value by the operating system.

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (yonko2) [inline Seminars/Sem.03/Pract.03/Task02.cpp:40]: Could validate console inputs

File: Seminars/Sem.03/Pract.03/Task02.cpp
Comment (yonko2) [inline Seminars/Sem.03/Pract.03/Task02.cpp:18]: Mark positive number fields as unsigned

**Pull request closed without merging**
